common/lib/xmodule/xmodule/js/src/video/04_video_control.js

```diff
@@ -149,11 +149,23 @@ function() {
         });
     }
 
+    var isTrackingEventSent = false;
+
     function updateVcrVidTime(params) {
         var endTime = (this.config.endTime !== null) ? this.config.endTime : params.duration;
         // in case endTime is accidentally specified as being greater than the video
         endTime = Math.min(endTime, params.duration);
-        this.videoControl.vidTimeEl.html(Time.format(params.time) + ' / ' + Time.format(endTime));
+        var elapsedTime = Time.format(params.time);
+        this.videoControl.vidTimeEl.html(elapsedTime + ' / ' + Time.format(endTime));
+        if (elapsedTime === '0:10') { // Track GTM only at a particular time
+            if (!isTrackingEventSent) { // preventing multiple GTM events
```

Comment:
Let's please add a brief reason for multiple calls here in the comment.

Reply:
done